src/components/RadioButton/__tests__/RadioButtonGroup.spec.tsx

```diff
@@ -43,26 +44,26 @@ describe('<RadioButtonGroup> that renders a group of radio buttons', () => {
         await user.click(buttons[0]);
 
         expect(onChangeChildMock).toHaveBeenCalled();
+        expect(buttons[1]).not.toHaveAttribute('onChange');
     });
 
     it('should pass onChange prop to children', async () => {
         const user = userEvent.setup();
         const onChangeGlobalMock = jest.fn();
-        view = render(
-            <RadioButtonGroup name="group_name">
-                <RadioButton onChange={onChangeGlobalMock} id="o1">
-                    Option 1
-                </RadioButton>
+        view.rerender(
+            <RadioButtonGroup name="group_name" onChange={onChangeGlobalMock}>
+                <RadioButton id="o1">Option 1</RadioButton>
                 <RadioButton id="o2">Option 2</RadioButton>
             </RadioButtonGroup>
         );
 
         const buttons = screen.getAllByRole('radio');
 
-        expect(buttons).toHaveLength(4);
+        expect(buttons).toHaveLength(2);
 
-        await user.click(buttons[2]);
+        await user.click(buttons[0]);
 
         expect(onChangeGlobalMock).toHaveBeenCalled();
+        expect(buttons[1]).not.toHaveAttribute('onChange');
```

Comment:
hhmm. This is not what should happen. Maybe this check is not a good one, if it passes both here and in the test above. 

Here, you are trying to see that the global handler is passed to the children and is triggered when you click on the radio button.

Reply:
Yes and check that it contains onChangeGlobalMock for onChange

Reply:
but the test say `not.toHaveAttribute('onChange')`.... 🤔